package.json

```diff
@@ -36,6 +36,9 @@
   },
   "main": "index.js",
   "devDependencies": {
+    "@types/chai": "^4.1.7",
```

Comment:
are these types used or can they be removed? We're not using typescript in the project

Reply:
Sorry, I use webstorm and it help me, because this IDE show me alerts when it's not understand the methods. How the IDE works with declaration of typescript, I included for remove the errors alert.

Reply:
Yes! We can remove!